frontend/src/breeding_form.tsx

```diff
@@ -241,6 +248,38 @@ export function BreedingForm({
     );
   };
 
+  const handleEditableBreedingUpdates = async (
+    breeding: Breeding,
+    breedingMatch: Breeding,
+    parentsUpdate: boolean
+  ) => {
+    const modifiedBreedingUpdates = modifyBreedingUpdates(
+      breeding,
+      breedingMatch
+    );
+    const newIndsNumber = checkBirthUpdate(
+      breedingMatch,
+      modifiedBreedingUpdates
+    );
+    const updatedBreeding = await updateBreeding(modifiedBreedingUpdates);
+    if (!!updatedBreeding) {
+      if (parentsUpdate) {
+        userMessage("Parningstillfället har uppdaterats.", "success");
+      } else {
+        userMessage(
+          "Klart. Parningen redan finns i systemet så föreldrarna har inte uppdaterats!",
+          "success"
```

Comment:
I think proper Swedish would be 
```suggestion
          "Klart. Parningen finns redan i systemet så föreldrarna har inte uppdaterats!",
          "success"
```
:)